xbmc/cores/VideoPlayer/DVDCodecs/Video/AMLCodec.cpp

```diff
@@ -1599,7 +1599,7 @@ bool CAMLCodec::OpenDecoder(CDVDStreamInfo &hints)
         am_private->gcodec.param = (void*)(EXTERNAL_PTS | SYNC_OUTSIDE);
       break;
   }
-  am_private->gcodec.param = (void *)((unsigned int)am_private->gcodec.param | (am_private->video_rotation_degree << 16));
+  am_private->gcodec.param = (void *)((unsigned long)am_private->gcodec.param | (am_private->video_rotation_degree << 16));
```

Comment:
sizeof(void*) is for sure not what you want.

2016-09-04 21:07 GMT+02:00 mapfau <notifications@github.com>:

> In xbmc/cores/VideoPlayer/DVDCodecs/Video/AMLCodec.cpp
> <https://github.com/xbmc/xbmc/pull/10401#discussion_r77457005>:
>
> > @@ -1599,7 +1599,7 @@ bool CAMLCodec::OpenDecoder(CDVDStreamInfo &hints)
> >          am_private->gcodec.param = (void*)(EXTERNAL_PTS | SYNC_OUTSIDE);
> >        break;
> >    }
> > -  am_private->gcodec.param = (void *)((unsigned int)am_private->gcodec.param | (am_private->video_rotation_degree << 16));
> > +  am_private->gcodec.param = (void *)((unsigned long)am_private->gcodec.param | (am_private->video_rotation_degree << 16));
>
> or size_t ?
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/xbmc/xbmc/pull/10401/files/7a085bc6b2496c88ec87efcd3758fa3e80bcbcee#r77457005>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ABCfHbCVSZgX1Y7RsJZfTFwMUY7kxNU9ks5qmxb0gaJpZM4J0mho>
> .
>



-- 
                   Key-ID:     0x1A995A9B
                   keyserver: pgp.mit.edu
==============================================================
Fingerprint: 4606 DA19 EC2E 9A0B 0157  C81B DA07 CF63 1A99 5A9B